pkg/tsdb/testdatasource/scenarios.go

```diff
@@ -652,7 +652,7 @@ func (s *Service) handleLogsScenario(ctx context.Context, req *backend.QueryData
 }
 
 func RandomWalk(query backend.DataQuery, model *simplejson.Json, index int) *data.Frame {
-	rand := rand.New(rand.NewSource(time.Now().UnixNano()))
+	rand := rand.New(rand.NewSource(time.Now().UnixNano() + int64(index)))
```

Comment:
This adds series index to the seed. I encountered a situation when several consequtive evaluations returned series with exactly the same points